docs/cmdline-opts/tls13-ciphers.md

```diff
@@ -2,16 +2,16 @@
 c: Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
 SPDX-License-Identifier: curl
 Long: tls13-ciphers
-Arg: <list>
+Arg: <list of ciphers>
```

Comment:
Since the option name includes "ciphers" I figured an optimization to keep this short is to not include "ciphers" again in the argument. The help output needs to be as short as possible.